code/game/machinery/scanner_gate.dm

```diff
@@ -175,6 +199,11 @@
 					beep = TRUE
 				if(H.nutrition >= detect_nutrition && detect_nutrition == NUTRITION_LEVEL_FAT)
 					beep = TRUE
+		if(SCANGATE_CONTRABAND)
+			for(var/obj/content in M.get_contents())
+				if(HAS_TRAIT(content, TRAIT_CONTRABAND))
```

Comment:
Are objects such as implants, roundstart tot gear or things stored in storage implants (if the implant itself isn't already) detected by this?

Doesn't this massively simplify being able to search for antags passively?